### PR TITLE
Qa for #1023

### PR DESCRIPTION
**Pull request by TheDauntless**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [x] Your contribution is written in the 2nd person (e.g. you)
- [x] Your contribution is written in an active present form for as much as possible.
- [x] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [x] Your contribution has proper formatted markdown and/or code
- [x] Any references to website have been formatted as [TEXT](URL “NAME”)
- [x] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #1023 

--
I've changed the perspective of the attack surface, since "Having unsafe input" is worded weirdly. Feel free to discuss :).

### CODE REVIEW

File: Document/0x05a-Platform-Overview.md [hunk 1]
```diff
@@ -579,19 +579,19 @@ Publishing an app is quite straightforward; the main operation is making the sig
...
+- Validate all input by the user in input fields.
+- Validate the content loaded inside a webview. See:
+  -  [Testing JavaScript execution in webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-javascript-execution-in-webviews "Testing JavaScript execution in webviews");
   - [Testing WebView Protocol Handlers](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#testing-webview-protocol-handlers "Testing WebView Protocol Handlers");
   - [Determining Whether Java Objects Are Exposed Through WebViews]("https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05h-Testing-Platform-Interaction.md#determining-whether-java-objects-are-exposed-through-webviews "Determining Whether Java Objects Are Exposed Through WebViews").
-- Having insecure responses from a server, or compromised responses by means of a man in the middle attack between the server and the mobile application. See:
+- Securely communicate with backend servers and is susceptible to man-in-the-middle attacks between the server and the mobile application. See:
```
Comment (commjoen) [inline Document/0x05a-Platform-Overview.md]: should be "and is not susceptible to" then right?

File: Document/0x05a-Platform-Overview.md [hunk 2]
```diff
@@ -579,19 +579,19 @@ Publishing an app is quite straightforward; the main operation is making the sig
...
   - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication");
   - [Android Network APIs](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x05g-Testing-Network-Communication.md#android-network-apis "Android Network APIs") .
-- Having insecure or compromised storage. See:
+- Securely stores all local data, or loads untrusted data from storage. See:
```
Comment (commjoen) [inline Document/0x05a-Platform-Overview.md]: should be "or sanitizes untrusted data from..."right?

File: Document/0x06a-Platform-Overview.md [hunk 3]
```diff
@@ -184,17 +184,17 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
-- Having unsafe input by the user to input fields.
-- Having unsafe input to a WebView by a user or by having insecure code loaded into the WebView. See:
+- Validate all input by the user in input fields.
+- Validate the content loaded inside a webview. See:
   -  [Testing iOS webviews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#testing-ios-webviews "Testing iOS webviews");
   - [Determining Whether Native Methods Are Exposed Through WebViews](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06h-Testing-Platform-Interaction.md#determining-whether-native-methods-are-exposed-through-webviews "Determining Whether Native Methods Are Exposed Through WebViews")
-- Having insecure responses from a server, or compromised responses by means of a man in the middle attack between the server and the mobile application. See:
+- Securely communicate with backend servers and is susceptible to man-in-the-middle attacks between the server and the mobile application. See:
```
Comment (commjoen) [inline Document/0x06a-Platform-Overview.md]: should be "and is not susceptible to" then right?

File: Document/0x06a-Platform-Overview.md [hunk 4]
```diff
@@ -184,17 +184,17 @@ The following APIs [require user permission](https://www.apple.com/business/docs
...
   - [Testing Network Communication](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x04f-Testing-Network-Communication.md#testing-network-communication "Testing Network Communication");
   - [iOS Network APIs](https://github.com/OWASP/owasp-mstg/blob/master/Document/0x06g-Testing-Network-Communication.md#ios-network-apis "iOS Network APIs") .
-- Having insecure or compromised storage. See:
+- Securely stores all local data, or loads untrusted data from storage. See:
```
Comment (commjoen) [inline Document/0x06a-Platform-Overview.md]: should be "or sanitizes untrusted data from..."right?

File: Document/0x05a-Platform-Overview.md (hunk 1 above)
Comment (TheDauntless) [reply Document/0x05a-Platform-Overview.md]: My bad!

**Comment by TheDauntless** [comment]
Hmm, no wait, because I reversed it at the top, they shouldn't be made negative.

> The Android application may be vulnerable to attack if it does not: 
...

I can rewrite it so that it's not negative?

**Pull request merged**